SPECIFICATION.md

```diff
@@ -0,0 +1,441 @@
+# Geospatial Widget (Kujaku) Specification
+
+## Table of Contents
+
+* [What is a Geospatial Widget?](#what-is-a-geospatial-widget)
+* [Specification](#specification)
+   * [Map Activity](#mapactivity)
+   * [How to create a Mapbox style with Kujaku configuration](#how-to-create-a-mapbox-style-with-kujaku-configuration)
+   * [Offline Maps Downloader Service](#offline-maps-downloader-service)
+   * [Kujaku Map View](#kujaku-map-view)
+   * [Helper Classes](#helper-classes)
+     * [MapBoxStyleHelper class](#1-mapboxstylehelper)
+     * [CoordinateUtils class](#2-coordinateutils)
+     * [MapBoxWebServiceApi](#3-mapboxwebserviceapi)
+
+
+## What is a Geospatial Widget?
+
+The Geospatial widget is an Android SDK designed to connect to the georegistry and other common geographical data sources like OSM and OGC servers. It also supports visualisation of geosptial data and inspection. It is designed to integrate into common mobile data collection tools used in global health.  It is expected to support the three elimination use cases including:
```

Comment:
Could you reword

>  It is expected to support the three elimination use cases including:

Maybe remove word elimination and three 


>  It is expected to support the below use cases among others:

Reply:
Cool, Reworded to `It is expected to support the following Malaria elimination use cases among others`